### PR TITLE
Improved the ORM Readme

### PR DESCRIPTION
**Pull request by lorenzo**
Added instructions for autoloading table and entity classes

### CODE REVIEW

File: src/ORM/README.md [hunk 1]
```diff
@@ -161,8 +161,76 @@ $cacheConfig = [
...
+When using the ORM as a standalone package, you are free to choose where to
+store these classes. For example, you could use the `Data` folder for this:
+
+```php
+<?php
+// in src/Data/Table/ArticlesTable.php
+namespace Acme\Data\Table;
+
+use Acme\Data\Entity\Article.php;
+use Acme\Data\Table\UsersTable.php;
```
Comment (garas) [inline src/ORM/README.md]: remove `.php`

Comment (lorenzo) [reply src/ORM/README.md]: Thanks for noticing

File: src/ORM/README.md [hunk 2]
```diff
@@ -161,8 +161,76 @@ $cacheConfig = [
    'prefix'    => 'orm_',
 ],
 Cache::setConfig('_cake_model_', $cacheConfig);
+
+## Creating Custom Table and Entity Classes
+
+By default, the Cake ORM uses the `\Cake\ORM\Table` and `\Cake\ORM\Entity` classes to
+intercat with the database. While using the default classes makes sense for
```
Comment (ADmad) [inline src/ORM/README.md]: Typo "intercat".

File: src/ORM/README.md [hunk 3]
```diff
@@ -161,8 +161,76 @@ $cacheConfig = [
...
+use Acme\Data\Entity\Article;
+use Acme\Data\Table\UsersTable;
+use Cake\ORM\Table;
+
+class ArticlesTable extends Table
+{
+    public function initialize()
+    {
+        $this->setEntityClass(Article::class);
+	$this->belongsTo('Users', ['className' => UsersTable::class]);
```
Comment (ADmad) [inline src/ORM/README.md]: Indentation is off.

File: src/ORM/README.md (hunk 2 above)
Comment (lorenzo) [reply src/ORM/README.md]: meow

File: src/ORM/README.md [hunk 4]
```diff
@@ -161,8 +161,76 @@ $cacheConfig = [
    'prefix'    => 'orm_',
 ],
 Cache::setConfig('_cake_model_', $cacheConfig);
+
```
Comment (ndm2) [inline src/ORM/README.md]: A code block closing ` ``` ` seems to be missing.

**Pull request merged**